src/components/Conversion/ConversionNOB.vue

```diff
@@ -24,10 +24,9 @@
           <v-form ref="form" lazy-validation>
             <v-textarea
               filled
-              persistent-hint
               class="mt-5"
               autocomplete="chrome-off"
-              label="Enter Nature of Business"
+              placeholder="Enter Nature of Business"
```

Comment:
Label always shows.

Placeholder only shows until user clicks in area (as desired by Roy).

Hint is the thing below the textarea; not used here.